pkg/apis/core/validation/validation.go

```diff
@@ -64,6 +64,7 @@ const isInvalidQuotaResource string = `must be a standard resource for quota`
 const fieldImmutableErrorMsg string = apimachineryvalidation.FieldImmutableErrorMsg
 const isNotIntegerErrorMsg string = `must be an integer`
 const isNotPositiveErrorMsg string = `must be greater than zero`
+const isInclusiveErrorMssg string = `must be between`
```

Comment:
Could you inline this constant to the test where it's used? I don't see any reason to have it.

Reply:
makes sense, fixed